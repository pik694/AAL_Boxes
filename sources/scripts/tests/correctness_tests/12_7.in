12
9 35 70
51 50 96
24 19 34
35 94 23
95 19 37
43 77 78
1 18 67
62 94 83
6 82 96
69 97 25
67 37 79
70 61 51

