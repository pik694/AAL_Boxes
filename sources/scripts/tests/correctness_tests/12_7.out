6 0 4 9 2 3 1 8 10 11 5 7 
1796165->1356692
